index.html

```diff
@@ -0,0 +1,624 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no" />
+    <!-- Bootstrap CSS -->
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/css/bootstrap.min.css"
+        integrity="sha384-JcKb8q3iqJ61gNV9KGb8thSsNjpSL0n8PARn9HuZOnIxN0hoP+VmmDGMN5t9UJ0Z" crossorigin="anonymous" />
+    <link rel="stylesheet" href="index.css" />
+    <title>RR C onference</title>
+</head>
+
+<body>
+    <!--upper navigation-->
+
+    <nav class="fixed-top">
+        <div class="row top_navigation">
+            <div class="col-9"></div>
+            <div class="col-3">
+                <ul>
+                    <li>
+                        <img src="./images/facebook (21).png" alt="facebook">
+                    </li>
+                    <li>
+                        <img src="./images/twitter.png" alt="twitter">
+                    </li>
+                    <li>English</li>
+                    <li>My page</li>
+                    <li>Logout</li>
+                </ul>
+            </div>
+            <div class="col-2"></div>
+        </div>
+    </nav>
+
+    <!--Lower navigation-->
+
+    <nav class="fixed-top">
+        <div class="row lower_navigation">
+            <div class="col-1"></div>
+            <div class="col-2 about_logo">
+                <a href=""><img src="./images/rrtb.avif" alt="" /></a>
+            </div>
+            <div class="col-1"></div>
+            <div class="col-8">
+                <ul>
+                    <li>
+                        <a class="follow" href="about.html">About</a>
+                    </li>
+                    <li>
+                        <a class="prog" href="##">Program</a>
+                    </li>
+                    <li>
+                        <a class="prog" href="##">Join</a>
+                    </li>
+                    <li>
+                        <a class="prog" href="##">Sponsor</a>
+                    </li>
+                    <li>
+                        <a class="prog" href="##">News</a>
+                    </li>
+                    <li>
+                        <a class="follow" href="##">Follow us</a>
+                    </li>
+                </ul>
```

Comment:
- [ ] Please reduce the gap between nave links to make it look identical to the one in the design guideline. 

Design Guideline:

<img width="1037" alt="Screenshot 2023-03-24 at 11 05 08 PM" src="https://user-images.githubusercontent.com/44798044/227617750-719cf5ab-1989-4919-a142-27d834f1ac8b.png">

Your design:

<img width="1506" alt="Screenshot 2023-03-25 at 12 07 35 AM" src="https://user-images.githubusercontent.com/44798044/227617796-7d46f1d8-fd01-4743-b7f5-defbbfecc60d.png">